src/java/fr/paris/lutece/plugins/oauth2/jwt/JWTParser.java

```diff
@@ -44,22 +44,36 @@
  */
 public interface JWTParser
 {
+ 
     /**
-     * Extract Json Web Token from the token
-     *
-     * @param token
-     *            The token
+     *  Extract Json Web Token from the token
+     * @param <T>
+     * @param strCompactJwt
```

Comment:
I whould use `unparsedJwt`. I don't think a JWT can be compact or not. a JWT is a compact representation of claims.